### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by cameracker**
This PR creates a security policy based off of recommendations from the OpenSSF Scorecard. 

A security policy may seem gratuitous, but its worthwhile to remember that this library was born from a security vulnerability on the repo from which it was forked, satori/go.uuid.  Our library is simple, but it doesn't mean its immune from vulnerabilities or security issues :)

With this security policy, we shoot for simplicity:

- Support latest, unless there's a very good reason to not. Our package is relatively easy to keep up to date, and we go through great pains to not break the API. As a result, we should be able to put forth an expectation of supporting latest.
- Lay out simple instructions for reporting a vulnerability
- Mention our cooperation with OpenSSF Scorecard, and make a nod to the fact that our actively maintained score may drop when there's just not much to do with the library.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Security Scorecard
+
+This project submits security [results](https://scorecard.dev/viewer/?uri=github.com/gofrs/uuid) to the [OpenSSF Scorecard](https://securityscorecards.dev/). 
+
+### Actively Maintained
+
+One heuristic these scorecards measure to gauge whether a package is safe for consumption is an "Actively Maintained" metric. Because this library implements UUIDs,
+it is very stable - there is not much maintenance required other than adding/updating newer UUID versions, keeping up to date with latest versions of Go, and responding
+to reported exploits. As a result, periods of low active maintance are to be expected.
```
Comment (dylan-bourque) [inline SECURITY.md:22]: "maintenance" is misspelled here

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+We support the latest version of this library. We do not guarantee support of previous versions. If a defect is reported, it will generally be fixed on the latest version 
+(provided it exists) irrespective of whether it was introduced to a prior version.
```
Comment (dylan-bourque) [inline SECURITY.md:6]: Should be "... introduced in a prior version."

**Review by dylan-bourque — APPROVED** [review]
Approved, with a couple grammatical nits

File: SECURITY.md (hunk 2 above)
Comment (cameracker) [inline SECURITY.md:6]:
```suggestion
(provided it exists) irrespective of whether it was introduced in a prior version.
```

File: SECURITY.md (hunk 1 above)
Comment (cameracker) [inline SECURITY.md:22]:
```suggestion
to reported exploits. As a result, periods of low active maintenance are to be expected. 
```

**Pull request merged**